### PR TITLE
Feature custom social

### LINKED ISSUE #91: Custom social services
Streamers have requested the ability to add custom services for `!social`. For example:

```
!social
Twitter: twitter.com/2CubedTech, GitHub: github.com/2Cubed

!social github
github.com/2Cubed
```

This makes sense, because we want to become service-agnostic, and our `!social` system currently depends on Beam.

### PR DESCRIPTION
**Pull request by Innectic**
Fixes #91

### CODE REVIEW

File: cactusbot/commands/magic/social.py [hunk 1]
```diff
@@ -13,32 +11,55 @@ class Social(Command):
...
+                if social.status == 200:
+                    data = await social.json()
+                    response.append(
+                        data["data"]["attributes"]["service"].title() + ': ')
+                    response.append(
+                        ("url", data["data"]["attributes"]["url"]))
+                    response.append(', ')
+                else:
+                    return "'{}' not found on the streamer's profile!".format(
+                        ', '.join(services))
```
Comment (RPiAwesomeness) [inline cactusbot/commands/magic/social.py]: Shouldn't `.join(services)` be `.join(service)`? Some of the services might not exist, just not this current one

Comment (RPiAwesomeness) [inline cactusbot/commands/magic/social.py]: Shouldn't .join(services) be .join(service)? Some of the services might not exist, just not this current one

File: cactusbot/commands/magic/social.py [hunk 2]
```diff
@@ -13,32 +11,55 @@ class Social(Command):
...
-        social_data = social_data["user"]["social"]
+                for service in data["data"]:
+                    response.append(
+                        service["attributes"]["service"].title() + ': ')
+                    response.append(("url", service["attributes"]["url"]))
+                    response.append(', ')
+                return MessagePacket(*response[:-1])
+            else:
+                return "'{}' not found on the streamer's profile!".format(
+                    ', '.join(services))
```
Comment (RPiAwesomeness) [inline cactusbot/commands/magic/social.py]: Same here

Comment (Innectic) [inline cactusbot/commands/magic/social.py]: Fixed

File: cactusbot/commands/magic/social.py (hunk 1 above)
Comment (Innectic) [inline cactusbot/commands/magic/social.py]: Fixed

**Pull request merged**